### PR TITLE
Remove DaoLookup, EntityGroup, SqlProfiler, original Hibernate DAOs, and associated code

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: sqlstore/src/main/resources/db/migration/V8900__hibernatize_everything.sql [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+) ENGINE=InnoDB;
+
+INSERT INTO ProjectOverview_Sample(projectOverview_overviewId, sample_sampleId)
+  SELECT parentId, entityId FROM EntityGroup JOIN EntityGroup_Elements ON EntityGroup.entityGroupId = EntityGroup_Elements.entityGroup_entityGroupId
+   WHERE entityType = 'uk.ac.bbsrc.tgac.miso.core.data.Sample' AND parentType = 'uk.ac.bbsrc.tgac.miso.core.data.impl.ProjectOverview';
+
+DROP TABLE EntityGroup_Elements;
+DROP TABLE EntityGroup;
+
+UPDATE Pool_Elements SET elementType = CASE elementType WHEN 'uk.ac.bbsrc.tgac.miso.core.data.impl.LibraryDilution' THEN 'L' WHEN 'uk.ac.bbsrc.tgac.miso.core.data.impl.emPCRDilution' THEN 'E' END;
```
Comment (djcooke) [inline sqlstore/src/main/resources/db/migration/V8900__hibernatize_everything.sql]: We want to make Libraries Poolable in the future I think... 'L' would be confusing in that case. Use longer String identifiers?

File: sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLPoolDAO.java [hunk 2]
```diff
@@ -526,18 +517,8 @@ public String mapRow(ResultSet rs, int pos) throws SQLException {
         esParams.addValue("elementType", d.getClass().getName());
 
         eInsert.execute(esParams);
+        // TODO Fixed in Hibernate
```
Comment (djcooke) [inline sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLPoolDAO.java]: Please leave original code commented out with a note that describes the necessary change. Code should not be removed entirely until class is Hibernatized to ensure that person doing that work understands the requirements and can verify they are met (and so reviewers can easier see the behaviour moved and unchanged)

File: sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLProjectDAO.java [hunk 3]
```diff
@@ -425,9 +417,7 @@ public long saveOverview(ProjectOverview overview) throws IOException {
       }
     }
 
-    if (overview.getSampleGroup() != null && !overview.getSampleGroup().getEntities().isEmpty()) {
-      entityGroupDAO.save(overview.getSampleGroup());
-    }
+    // TODO Hibernate ProjectOverview properly
```
Comment (djcooke) [inline sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLProjectDAO.java]: Leave original code commented out with note

**Comment by apmasell** [comment]
fixed.

**Pull request merged**